src/preamble.js

```diff
@@ -1117,7 +1117,7 @@ function createWasm() {
 #endif
 }
 
-// Globals used by JS i64 conversions
+// Globals used by JS i64 conversions (see makeSetValue)
```

Comment:
I'm not very familiar with this code, but this file doesn't need `#if SAFE_HEAP` guard as in preamble_minimal.js?

Reply:
`preamble_minimal.js` is tries to include stuff only when absolutely necessary.   It doesn't have very many users at this point.   The normal `preamble.js` is more conservative and includes these in all cases since most programs end up needing them.